actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -85,6 +85,9 @@ def controller_class
       end
     end
 
+    # Returns true if the request has a header matching the given key parameter.
+    #
+    #    request.key? :ip_spoofing_check # => true
```

Comment:
The version with a space is correct. There's over 3000 instances of that in our documentation, but just 1 of no space.